libs/email/components/content.tsx

```
@@ -2,33 +2,125 @@ import { Button, Card } from '@/email'
                     <td className="text-center">
+                      <pre className="text-sm text-[#e6e6e6] font-mono text-left">
+                        {body.text}
+                      </pre>
                       {body.buttonLink && (
-                        <a href={body.buttonLink}>
-                          <Button>
-                            <Text className="m-0 p-0">{body.buttonText}</Text>
-                          </Button>
-                        </a>
+                        <>
+                          <a href={body.buttonLink} target="_blank" className="block">
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)